homeassistant/components/nanoleaf/light.py

```diff
@@ -108,6 +108,7 @@ class NanoleafLight(LightEntity):
 
     def __init__(self, light, name):
         """Initialize an Nanoleaf light."""
+        self._unique_id = f"nanoleaf-{light.model}-{light.serialNo}"
```

Comment:
- Is the serial number not unique by itself?
- There is no need to prefix unique IDs with the domain

Reply:
* I didn't want to assume anything about Nanoleaf's numbering structure; unfortunately I only have multiple of their older Nanoleaf Panels and none of the newer product line so I cannot directly compare.
* Removed.